web/blueprints/finance/__init__.py

```diff
@@ -172,9 +188,12 @@ def bank_accounts_import():
                     bank_account.iban)
                 )
             start_date = form.start_date.data
-            statement = fints.get_statement(acc, start_date, date.today())
+            statement, with_error = fints.get_filtered_transactions(acc, start_date, date.today())
             flash(
                 "Transaktionen vom {} bis {}.".format(start_date, date.today()))
+            if len(with_error) > 0:
+                flash("{} Statements enthielten fehlerhafte Daten und müssen vor dem Import manuell korrigiert werden.".format(len(with_error)), 'error')
```

Comment:
Pretty sure that's long enough to violate the style guidelines for line length (90 soft limit, 100 hard limit) as it even wraps in the github diff view.